openmdao/core/system.py

```diff
@@ -3489,6 +3489,11 @@ def list_inputs(self,
         else:
             values = val
 
+        if self._problem_meta['setup_status'] == 2:
+            issue_warning("Calling `list_inputs` before `final_setup` or `run_model` will only "
```

Comment:
I would just mention 'final_setup' and wouldn't include 'run_model'.  'final_setup' is also called by 'run_driver', 'check_partials' and 'check_totals', but we probably don't want to mention them all, so just mentioning 'final_setup' seems cleaner.

Reply:
Fixed